features/concurrent-user-sharing.feature

```diff
@@ -0,0 +1,18 @@
+Feature: concurrent user sharing
+  As a user
+  I want to share their resource concurrently to different multiple users
+
+  @concurrent
+  Scenario: users make concurrent sharing to each other
+    Given user "admin" has logged in with password "admin"
+    And user "admin" has created a personal space with the alias "Admin Home"
+    And user "admin" has uploaded a file "testfile1.txt" with content "concurrent sharing" in the home directory with the alias "testfile1"
+    And user "admin" has uploaded a file "testfile2.txt" with content "concurrent sharing" in the home directory with the alias "testfile2"
+    When user "admin" shares a file "testfile1.txt" with the following users concurrently
```

Comment:
I didn't find how create new users. I don't know if it's possible to do it here.
as a trick to increase a shares count can also be shared with demo groups

Reply:
CC @butonic. If you could provide some details regarding the creation of the users.

Reply:
we cannot create more users. @SagarGi please add all existing group and we can merge PR

Reply:
@ScharfViktor sure!